packages/pack-agent-security/test/fixtures/bad-process-spawn.ts

```
@@ -0,0 +1,29 @@
+// Fixture: the #1486 rule MUST fire on every line below.
+// Each line is a canonical attack-pattern call site in a non-build path.
+/* eslint-disable @typescript-eslint/no-require-imports */
+/* eslint-disable import/no-commonjs */
```

ESLint

[error] Definition for rule 'import/no-commonjs' was not found.

(import/no-commonjs)